### ISSUE TITLE
Console Error - InventoryClickEvent

### ISSUE THREAD

**Issue by JohZant** [JohZant/GUICommands#15]
Just saw this come up in console.

```
[10:40:55] [Server thread/ERROR]: Could not pass event InventoryClickEvent to GUICommands v1.0.0
org.bukkit.event.EventException: null
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:306) ~[spigot.jar:git-Spigot-4bd94dc-c765646]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[spigot.jar:git-Spigot-4bd94dc-c765646]
	at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:500) [spigot.jar:git-Spigot-4bd94dc-c765646]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:485) [spigot.jar:git-Spigot-4bd94dc-c765646]
	at net.minecraft.server.v1_12_R1.PlayerConnection.a(PlayerConnection.java:1889) [spigot.jar:git-Spigot-4bd94dc-c765646]
	at net.minecraft.server.v1_12_R1.PacketPlayInWindowClick.a(SourceFile:33) [spigot.jar:git-Spigot-4bd94dc-c765646]
	at net.minecraft.server.v1_12_R1.PacketPlayInWindowClick.a(SourceFile:10) [spigot.jar:git-Spigot-4bd94dc-c765646]
	at net.minecraft.server.v1_12_R1.PlayerConnectionUtils$1.run(SourceFile:13) [spigot.jar:git-Spigot-4bd94dc-c765646]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_161]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_161]
	at net.minecraft.server.v1_12_R1.SystemUtils.a(SourceFile:46) [spigot.jar:git-Spigot-4bd94dc-c765646]
	at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:748) [spigot.jar:git-Spigot-4bd94dc-c765646]
	at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:406) [spigot.jar:git-Spigot-4bd94dc-c765646]
	at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:679) [spigot.jar:git-Spigot-4bd94dc-c765646]
	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:577) [spigot.jar:git-Spigot-4bd94dc-c765646]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_161]
Caused by: java.lang.NullPointerException
	at com.arch.guicommands.GUICommandsListener.lambda$onInventoryClick$1(GUICommandsListener.java:101) ~[?:?]
	at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:174) ~[?:1.8.0_161]
	at java.util.ArrayList$ArrayListSpliterator.tryAdvance(ArrayList.java:1359) ~[?:1.8.0_161]
	at java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:126) ~[?:1.8.0_161]
	at java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:498) ~[?:1.8.0_161]
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:485) ~[?:1.8.0_161]
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471) ~[?:1.8.0_161]
	at java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:152) ~[?:1.8.0_161]
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:1.8.0_161]
	at java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:464) ~[?:1.8.0_161]
	at com.arch.guicommands.GUICommandsListener.onInventoryClick(GUICommandsListener.java:102) ~[?:?]
	at sun.reflect.GeneratedMethodAccessor187.invoke(Unknown Source) ~[?:?]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_161]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_161]
	at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:302) ~[spigot.jar:git-Spigot-4bd94dc-c765646]
	... 15 more
```

**Comment by JohZant** [comment]
This is on v1.2.0

I just noticed that it says Version 1.0.0. I must have forgotten to update this to 1.2.0 in the plugin.yml. I'll make sure that 1.3.0 is updated correctly.

**Comment by JohZant** [comment]
I believe this error was due to an old plugin version. Not changing the version number in plugin.yml has not allowed me to see what version could have been the issue.

Closing for now.

**Comment by JohZant** [comment]
Looks the same. This will need to be investigated.

```
27.04 23:57:16 [Server] ERROR Could not pass event InventoryClickEvent to GUICommands v1.3.0
27.04 23:57:16 [Server] INFO org.bukkit.event.EventException: null
27.04 23:57:16 [Server] INFO at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:306) ~[spigot.jar:git-Spigot-4bd94dc-fb414fe]
27.04 23:57:16 [Server] INFO at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[spigot.jar:git-Spigot-4bd94dc-fb414fe]
27.04 23:57:16 [Server] INFO at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:500) [spigot.jar:git-Spigot-4bd94dc-fb414fe]
27.04 23:57:16 [Server] INFO at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:485) [spigot.jar:git-Spigot-4bd94dc-fb414fe]
27.04 23:57:16 [Server] INFO at net.minecraft.server.v1_12_R1.PlayerConnection.a(PlayerConnection.java:1889) [spigot.jar:git-Spigot-4bd94dc-fb414fe]
27.04 23:57:16 [Server] INFO at net.minecraft.server.v1_12_R1.PacketPlayInWindowClick.a(SourceFile:33) [spigot.jar:git-Spigot-4bd94dc-fb414fe]
27.04 23:57:16 [Server] INFO at net.minecraft.server.v1_12_R1.PacketPlayInWindowClick.a(SourceFile:10) [spigot.jar:git-Spigot-4bd94dc-fb414fe]
27.04 23:57:16 [Server] INFO at net.minecraft.server.v1_12_R1.PlayerConnectionUtils$1.run(SourceFile:13) [spigot.jar:git-Spigot-4bd94dc-fb414fe]
27.04 23:57:16 [Server] INFO at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_161]
27.04 23:57:16 [Server] INFO at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_161]
27.04 23:57:16 [Server] INFO at net.minecraft.server.v1_12_R1.SystemUtils.a(SourceFile:46) [spigot.jar:git-Spigot-4bd94dc-fb414fe]
27.04 23:57:16 [Server] INFO at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:748) [spigot.jar:git-Spigot-4bd94dc-fb414fe]
27.04 23:57:16 [Server] INFO at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:406) [spigot.jar:git-Spigot-4bd94dc-fb414fe]
27.04 23:57:16 [Server] INFO at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:679) [spigot.jar:git-Spigot-4bd94dc-fb414fe]
27.04 23:57:16 [Server] INFO at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:577) [spigot.jar:git-Spigot-4bd94dc-fb414fe]
27.04 23:57:16 [Server] INFO at java.lang.Thread.run(Thread.java:748) [?:1.8.0_161]
27.04 23:57:16 [Server] INFO Caused by: java.lang.NullPointerException
```

**Comment by JohZant** [comment]
This just cracked it again. 

```
29.04 22:24:00 [Server] ERROR Could not pass event InventoryClickEvent to GUICommands v1.3.1
29.04 22:24:00 [Server] INFO org.bukkit.event.EventException: null
29.04 22:24:00 [Server] INFO at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:306) ~[spigot.jar:git-Spigot-4bd94dc-fb414fe]
29.04 22:24:00 [Server] INFO at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[spigot.jar:git-Spigot-4bd94dc-fb414fe]
29.04 22:24:00 [Server] INFO at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:500) [spigot.jar:git-Spigot-4bd94dc-fb414fe]
29.04 22:24:00 [Server] INFO at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:485) [spigot.jar:git-Spigot-4bd94dc-fb414fe]
29.04 22:24:00 [Server] INFO at net.minecraft.server.v1_12_R1.PlayerConnection.a(PlayerConnection.java:1889) [spigot.jar:git-Spigot-4bd94dc-fb414fe]
29.04 22:24:00 [Server] INFO at net.minecraft.server.v1_12_R1.PacketPlayInWindowClick.a(SourceFile:33) [spigot.jar:git-Spigot-4bd94dc-fb414fe]
29.04 22:24:00 [Server] INFO at net.minecraft.server.v1_12_R1.PacketPlayInWindowClick.a(SourceFile:10) [spigot.jar:git-Spigot-4bd94dc-fb414fe]
29.04 22:24:00 [Server] INFO at net.minecraft.server.v1_12_R1.PlayerConnectionUtils$1.run(SourceFile:13) [spigot.jar:git-Spigot-4bd94dc-fb414fe]
29.04 22:24:00 [Server] INFO at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_161]
29.04 22:24:00 [Server] INFO at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_161]
29.04 22:24:00 [Server] INFO at net.minecraft.server.v1_12_R1.SystemUtils.a(SourceFile:46) [spigot.jar:git-Spigot-4bd94dc-fb414fe]
29.04 22:24:00 [Server] INFO at net.minecraft.server.v1_12_R1.MinecraftServer.D(MinecraftServer.java:748) [spigot.jar:git-Spigot-4bd94dc-fb414fe]
29.04 22:24:00 [Server] INFO at net.minecraft.server.v1_12_R1.DedicatedServer.D(DedicatedServer.java:406) [spigot.jar:git-Spigot-4bd94dc-fb414fe]
29.04 22:24:00 [Server] INFO at net.minecraft.server.v1_12_R1.MinecraftServer.C(MinecraftServer.java:679) [spigot.jar:git-Spigot-4bd94dc-fb414fe]
29.04 22:24:00 [Server] INFO at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:577) [spigot.jar:git-Spigot-4bd94dc-fb414fe]
29.04 22:24:00 [Server] INFO at java.lang.Thread.run(Thread.java:748) [?:1.8.0_161]
29.04 22:24:00 [Server] INFO Caused by: java.lang.NullPointerException
```

Not in the method, as was previously thought.

**Comment by JohZant** [comment]
Should be fixed as of v1.3.2.
